test/indexer/test_indexer.py

```diff
@@ -2113,6 +2114,54 @@ def test_disallow_manifest_column_joiner(self):
         with self.assertRaisesRegex(RequirementError, "'||' is disallowed"):
             self._index_bundle(bundle)
 
+    def test_bionetwork_fields(self):
+        bundle_fqid = self.bundle_fqid(uuid='3ac62c33-93e1-56b4-b857-59497f5d942d',
+                                       version='2021-01-01T00:00:00.000000Z')
+        bundle = self._load_canned_bundle(bundle_fqid)
+        # The case of an empty array is covered elsewhere since the older canned
```

Comment:
```suggestion
        # The case of an empty array is covered elsewhere since the older canned
```

Where is it covered? Absence and emptiness are different cases.

Reply:
If we distinguish absence and emptiness, then it was not covered. I conflated the two because I knew that the implementation used an array as the default value when the property was absent.